category.php

```diff
@@ -1,12 +1,8 @@
 <?php get_header(); ?>
 
-<div class="hero-container">
-  <div class="hero-image" style="background-image: url(<?php echo get_template_directory_uri(); ?>/images/bg/contact.jpg);">
-    <div class="container">
-      <div class="hero-heading">
-        <h1> Projects </h1>
-      </div>
-    </div>
+<div class="hero">
```

Comment:
No, it currently is not fine as a partial, because several of the pages need customized h1 tags that are part of page-content and it doesn't make sense to move those to some other file for capture and display.  It's 5 lines of code, and isn't even particularly ugly.